cmd/state-installer/installer.go

```diff
@@ -24,63 +26,58 @@ import (
 )
 
 func main() {
-	exitCode := run()
-	if exitCode != 0 {
-		os.Exit(exitCode)
+	if !_run() {
+		os.Exit(1)
 	}
 }
 
-func run() int {
+func _run() bool {
```

Comment:
This feels really dirty just to facilitate a `defer`, which we don't HAVE to use. It's there for your convenience, so if it's not convenient just don't use it.

I think what you want is for the main function to call rollbar.Close() before you os.Exit(), either by putting all that in a central exit function or just by actually calling `rollbar.Close()` right before the exit. 

I'm not married to that suggestion, I just want a solution that doesn't make you scratch your head when you look at it.